falcon/errors.py

```diff
@@ -46,16 +46,46 @@ def on_get(self, req, resp):
 class HTTPBadRequest(HTTPError):
     """400 Bad Request.
 
-    The request could not be understood by the server due to malformed
-    syntax. The client SHOULD NOT repeat the request without
-    modifications. (RFC 2616)
+    The server cannot or will not process the request due to something
+    that is perceived to be a client error (e.g., malformed request
+    syntax, invalid request message framing, or deceptive request
+    routing).
+
+    (See also: RFC 7231, Section 6.5.1)
 
     Args:
         title (str): Error title (e.g., 'TTL Out of Range').
         description (str): Human-friendly description of the error, along with
             a helpful suggestion or two.
-        kwargs (optional): Same as for ``HTTPError``.
 
+    Keyword Args:
```

Comment:
I also added these sections to be more clear which kwargs can be used, and which are required or already filled in.